support-frontend/assets/pages/paper-subscription-checkout/paperSubscriptionCheckoutReducer.js

```diff
@@ -228,20 +231,25 @@ function initReducer(initialCountry: IsoCountry, productInUrl: ?string, fulfillm
   const user = getUser(); // TODO: this is unnecessary, it should use the user reducer
   const { productPrices } = window.guardian;
 
+  const product = getInitialProduct(productInUrl, fulfillmentInUrl);
+  const days: Date[] = product.fulfilmentOption === HomeDelivery
```

Comment:
I see that we're already calculating this list of days here: https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/pages/paper-subscription-checkout/components-checkout/checkoutForm.jsx#L88

Can anyone think of a way to avoid doing this twice or is this OK?

Reply:
maybe @damien-shaw-guardian @rupertbates ?